scripts/modules/MsIndustryLinks/azureDeploymentPackage/AzureDeploymentPackage.psm1

```diff
@@ -0,0 +1,329 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+<#
+    .Synopsis
+    Generates ARM templates that deploys the Azure resources
+    required for an Industry Link.
+
+    .Description
+    Generates ARM templates that deploys the Azure resources such
+    as storage account, event hub, connections and Logic Apps
+    required for an Industry Link.
+
+    .Parameter WorkflowConfigFile
+    The workflow configuration file that defines the trigger, the data
+    source, the data sink and any transformations that will be applied.
+
+    .Parameter TemplateDirectory
+    The path to the workflow templates required for your Industry Link. This
+    should contain at least one workflow template.
+
+    .Parameter OutputDirectory
+    The directory path where the ARM templates will be saved.
+
+    .Example
+    # Generate ARM templates from Logic App workflow templates
+    New-AzureDeploymentPackage -WorkflowConfigFile workflow.json -TemplateDirectory templates -OutputDirectory output
+#>
+function New-AzureDeploymentPackage {
```

Comment:
Will we be adding in some scripts to automate the deployment of the assets generated in this module?
Without these and documentation, I am finding this hard to set up and test.

Reply:
Yes, documentation is added in another PR.